### DISCUSSION TITLE
What platform do you prefer for having open-source community chat?

### DISCUSSION THREAD

**Discussion by erogol** [coqui-ai/TTS#895] [General]
🗳 Different platforms, pros, and cons. It is difficult to choose and settle your community.

it'd be interesting to see what people think about it.

1️⃣ Discourse
2️⃣ Element IO (Matrix)
3️⃣ Gitter
4️⃣ Discord
5️⃣ Slack

Please comment below. 

(Let me know if you know better ways to run Polls on Github too 😄)

**Comment by KakoozaJerry** [comment]
2
